_pages/verify-receive-address/advanced.md

```diff
@@ -38,11 +38,35 @@ We can improve this by using an offline/dedicated machine, where we install only
 While this will help protect you against malware on your computer, you are still at risk from an [Evil maid attack](https://en.wikipedia.org/wiki/Evil_maid_attack) with physical access to your computer (or paper printout).
 Such an "evil maid" could tamper with this software/printout to instead show bitcoin addresses that they control, and trick you into receiving a deposit on their address.
 
-#### Option A: Dedicated Machine
-TODO: add instructions for inputting extended public keys & paths (no private keys/seeds) running Electrum, Sparrow, Caravan, or some CLI script.
+#### Option A: Offline/Dedicated Machine
+You will set up the multisig in the software wallet of your choice  by importing the extended public keys for all N seeds from your quorum.
 
-It is recommended to use an eternally quarantined machine, meaning that it is never again connected to the internet and not used for any other purpose.
-That way, the attack surface is reduced and you are not at risk of malware on an ongoing basis.
+It is recommended to use an eternally quarantined machine, meaning that it is never again connected to the internet and not used for any other purpose. That way, the attack surface is reduced and you are not at risk of malware on an ongoing basis. 
+
+Alternatively, you can use Tails, which you might already have used when [setting up your Paper Wallet](https://btcguide.github.io/setup-wallets/paper-advanced).
+
+1. [Verify](https://btcguide.github.io/setup-wallets/coordinate-multisig-advanced) that all the Zpub/xpub keys match on both the Cobo and the Coldcard
+2. Reboot and run Tails
+
+##### Electrum
+
+3. Open Electrum, create a new Multi-signature wallet
+4. Move the sliders to use "From 3 cosigners", "Require 2 signatures"
+5. Add cosigner 1, by Entering cosigner "key". Do not enter any "seed"
+6. Type in the first Zpub that is [shown](https://btcguide.github.io/setup-wallets/coordinate-multisig-advanced) on either your Cobo or Coldcard. Click Next
+7. Repeat for cosigners 2 and 3
+8. You can skip setting a password, since this wallet will be deleted when Tails shuts down anyway
+9. The wallet is now created. Go to the Addresses tab to verify the addresses. If you cannot see the Addresses tab, use the menu item for View -> Show Addresses
```

Comment:
I wrote a PR into specter-desktop to export a watch-only wallet into Electrum that saves you steps 3-7, the guide should use that!

https://github.com/cryptoadvance/specter-desktop/pull/530

Reply:
Ahhh yes nice, that would save typing in those Zpubs. That was quite tedious for me.

However, I keep getting stuck in my head about this... isn't the whole point not to trust the computer which may be infected with malware? So if we trust the Specter export and import into Electrum, couldn't that be compromised?

Reply:
Ya, we still want to verify it. But the best time to verify is once the zpubs are loaded into electrum. So the tool is a useful way to export them, but you still want to compare against the hardware wallets (not specter-desktop).

Reply:
Oh I think I get it. You mean we don't have to trust the specter export, because if its compromised, then Electrum will generate bad addresses anyway. And the whole point is that we then want to cross check the addresses in Electrum with the addresses in the HWWs.

Reply:
Exactly. If you want to be extra cautious you can (re)verify at each step, but it's the final step that actually matters.